packaging/debian-control/libnetwork-prerm.sh

```diff
@@ -6,6 +6,5 @@ set -e
 /etc/init.d/libnetwork stop
 
 update-rc.d -f  libnetwork remove
-
+rm -rf /opt/pg/libnetwork/
```

Comment:
I would recommend to only remove the related files i.e. config.ini and libnetwork binary. As libnetwork directory may contain some files that are not to be removed in reinstall of libnetwork.